lib/view_component/base.rb

```diff
@@ -205,6 +205,44 @@ def content_evaluated?
     class << self
       attr_accessor :source_location, :virtual_path
 
+      # Find sidecar files for the given extensions.
+      #
+      # The provided array of extensions is expected to contain
+      # strings starting without the "dot", example: `["erb", "haml"]`
+      def sidecar_files(extensions)
```

Comment:
I'm not sure that we want to make this part of the public API just yet. Thoughts on prefixing this with an `_` and making it private, or moving it back to the compiler?